frontend/styleguide.config.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
   // propsParser: require('react-docgen-typescript').withCustomConfig('./tsconfig.json').parse,
 
   exampleMode: 'expand',
+  skipComponentsWithoutExample: true,
```

Comment:
Why was this added?

Reply:
So that a seperate section isn't included for DraggableContext.js as this is adequately documented in Draggable.js & Draggable.md

Reply:
So, it strikes me that ignoring components without documentation comments would be better.

Or, be explicit and use [ignore](https://github.com/styleguidist/react-styleguidist/blob/master/docs/Cookbook.md#how-to-exclude-some-components-from-style-guide).